src/one_dragon/base/config/user_config.py

```
@@ -0,0 +1,231 @@
+            config=self,
+            field=prop,
+            getter_convert=getter_convert,
+            setter_convert=setter_convert
+        )
+
+    def _get_key(self, module_name) -> Optional[str]:
+        if self.is_mock:
+            return None
+        path = module_name
+        if self.sub_dir:
+            path = '/'.join(self.sub_dir + [module_name])
```

Ruff (0.13.1)

Consider `[*self.sub_dir, module_name]` instead of concatenation

Replace with `[*self.sub_dir, module_name]`

(RUF005)